c2corg_ui/static/js/gpxupload.js

```diff
@@ -59,6 +61,60 @@ app.GpxUploadController = function($scope) {
 
 
 /**
+ *
+ * @param {Array.<ol.Feature>} features - invalid features that have to be checked
+ * @returns {Array.<ol.Feature>} features - this geometry is valid
+ * @private
+ */
+app.GpxUploadController.prototype.validatedFeatures_ = function(features) {
+  for (var i = 0; i < features.length; i++) {
+    /**
+     *
+     * @type {ol.geom.Geometry}
+     */
+    var geom = /**@type{ol.geom.Geometry}*/ (features[i].getGeometry());
+
+    /**
+     *
+     * @type {ol.geom.GeometryType}
+     */
+    var geomType = geom.getType();
+
+    // TODO validate LineString? pass now - no problem we know about?
+    // if (geomType === 'LineString') {
+    // }
+
+    if (geomType === 'MultiLineString') {
```

Comment:
Why not rather
```
if (geom instanceof ol.geom.MultiLineString)
```
as in https://github.com/c2corg/v6_ui/blob/7b5e5e79fd7d5a7bd36c0e41cac7e1ee158613dd/c2corg_ui/static/js/map/simplify.js#L183
https://github.com/c2corg/v6_ui/blob/0bf5dd38cd30b5aa94debbbf61737d1bf68f6f75/c2corg_ui/static/js/utils.js#L312
https://github.com/c2corg/v6_ui/blob/ab1514dc90c4ecf6395faa0326a03e1f18fdbb80/c2corg_ui/static/js/editing/documentediting.js#L433
?